MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,71 +2,172 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    
+    @IBOutlet private weak var imageView: UIImageView!
+    
+    @IBOutlet private weak var counterLabel: UILabel!
+    
+    @IBOutlet private weak var textLabel: UILabel!
+    
+    private var correctAnswers:Int = 0
+    
+    private var currentQuestionIndex: Int = 0
+    var currentQuestion: QuizQuestion {
+        get{
```

Comment:
перед фигурной скобкой пробел всегда, после либо тоже пробел либо с новой строки с табом вперед
```suggestion
        get {
```